src/Uno.UI/UI/Xaml/Media/Animation/DoubleAnimation.cs

```diff
@@ -342,15 +342,19 @@ private void OnEnd()
 
 			if (FillBehavior == FillBehavior.HoldEnd)//Two types of fill behaviors : HoldEnd - Keep displaying the last frame
 			{
-				HoldValue();
+				// We set the final value using the "Local" precedence
```

Comment:
so this is the key change? the PR description mentions setting the value with animations precedence but here it is local precedence?